examples/SAP-to-Database/main.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/sql;
```

Comment:
reformat this file to arrange the import order.

Reply:
Improved here [Improve SAP to DB example](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/23/commits/e6de654368da7d53a4bfc51ded0122365908f2b2)